dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ConnectorPropertyWithDefault.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022-2023 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.connector;
+
+import javax.annotation.Nullable;
+
+public interface ConnectorPropertyWithDefault extends ConnectorProperty {
```

Comment:
Optional: I would very slightly suggest avoiding the name containing the `With` word, unless there is no other option. In this case I don't have any good ideas, but I would rename the interface to `RichConnectorProperty`.

Reply:
the prefix `Rich` seems a bit vague to me. I also don't like the current naming but could not find a better one